routing/cross_mwm_connector.hpp

```diff
@@ -145,24 +145,14 @@ class CrossMwmConnector final
   void GetEdgeList(Segment const & segment, bool isOutgoing, std::vector<SegmentEdge> & edges) const
   {
     auto const & transition = GetTransition(segment);
-    if (isOutgoing)
+    // Ingoing edges listing is not supported for leaps because we do not have enough information
+    // to calculate |segment| weight. See https://jira.mail.ru/browse/MAPSME-5743 for details.
+    CHECK(isOutgoing, ("Ingoing edges listing is not supported for CrossMwmConnector."));
```

Comment:
Для чего тогда этот параметр если он всегда true?

Reply:
можно убрать, тогда в CrossMwmGraph будет как-то так:
```C++
void CrossMwmGraph::GetEdgeList(Segment const & s, bool isOutgoing, vector<SegmentEdge> & edges)
{
  CHECK(IsTransition(s, !isOutgoing), ("The segment is not a transition segment. IsTransition(", s,
                                       ",", !isOutgoing, ") returns false."));
  edges.clear();

  if (TransitGraph::IsTransitSegment(s)) && TransitCrossMwmSectionExists(s.GetMwmId()))
  {
    // Ingoing edges listing is not supported for leaps because we do not have enough information
    // to calculate |segment| weight. See https://jira.mail.ru/browse/MAPSME-5743 for details.
    CHECK(isOutgoing, ("Ingoing edges listing is not supported."));
    m_crossMwmTransitGraph.GetOutgoingEdgeList(s, edges);
  }
  else if (CrossMwmSectionExists(s.GetMwmId()))
  {
    // Ingoing edges listing is not supported for leaps because we do not have enough information
    // to calculate |segment| weight. See https://jira.mail.ru/browse/MAPSME-5743 for details.
    CHECK(isOutgoing, ("Ingoing edges listing is not supported."));
    m_crossMwmIndexGraph.GetOutgoingEdgeList(s, edges);
  }
  else
  {
    m_crossMwmOsrmGraph.GetEdgeList(s, isOutgoing, edges);
  }
}
```
но после выпиливания OSRM оно упростится и можно будет тоже избавиться от isOutgoing

Reply:
Мое имхо что если здесь уже есть знание о том что только исходящие ребра нужны, то можно и метод переименовать на GetOutgoingEdges, и параметр выпилить, чтобы это знание не размазывалось по системе.

Reply:
сделала